lib/confirmation_sender.rb

```diff
@@ -0,0 +1,5 @@
+module ConfirmationSender
```

Comment:
Laszlo I'm open to refactoring this but I'd prefer to keep it as a module as opposed to a service class.  What do you guys think? @cews7 @ethanbennett